.github/actions/python-requirements/requirements-friesian-python-ut-spark3.txt

```diff
@@ -26,3 +26,4 @@ fsspec==2021.9.0
 scikit-optimize==0.8.1
 pandas==1.1.5
 sentence-transformers
+wheel
```

Comment:
why do we need to install wheel?

Reply:
Some github action test will occur error without wheel.
![a8caedca-6c8d-44c0-9eea-be5ad94c8f61](https://user-images.githubusercontent.com/78460824/199915747-df6bf15c-2142-4bdd-99b8-f1907845a026.jpg)

Reply:
ok